docs/WritingTests.asciidoc

```diff
@@ -1408,3 +1408,13 @@ state of the specified git refspec. For example the asset caching
 functionality will override the +PRODUCTDIR+ variable. The above method can
 still be used in this case in the common case that the test schedule does not
 need to be changed or in case a custom schedule is defined by +SCHEDULE+.
+
+==Controling list of log files to upload
+
+During every job run worker produces various log files.
+By specifying +AUTOINST_UPLOAD_FROM_POOL+ variable with comma separated list of filenames
+it is possible to control which files should be uploaded to job results. Default set of files
```

Comment:
your line wrapping is off, please fix that

Reply:
not so easy to do when you not using awesome vim , but MS Visual Studio Code :D but I hope now all is fine